lib/chef/provisioning/docker_driver/driver.rb

```diff
@@ -105,7 +110,8 @@ def build_container(machine_spec, machine_options)
         target_repository = 'chef'
         target_tag = machine_spec.name
 
-        image = find_image(target_repository, target_tag)
+        # check if target image exists, if not try to look up for source image.
```

Comment:
I would rather this comment explain why we should use the source_tag if the target_tag doesn't return an image